File: tkpdabstraction/src/main/java/com/tokopedia/abstraction/common/data/model/request/GraphqlRequest.java
package com.tokopedia.abstraction.common.data.model.request;

import com.google.gson.annotations.SerializedName;

import java.util.HashMap;

/**
 * @author by milhamj on 26/02/18.
 */

public class GraphqlRequest {
    @SerializedName("query")
    public String query;

    @SerializedName("variables")
    public HashMap<String, Object> variables;

    public GraphqlRequest(String query, HashMap<String, Object> variables) {
        this.query = query;
        this.variables = variables;
    }
}

File: features/content/kol/src/main/java/com/tokopedia/kol/common/data/model/request/GraphqlRequest.kt
package com.tokopedia.kol.common.data.model.request

import com.google.gson.annotations.SerializedName

/**
 * @author by milhamj on 26/02/18.
 */
data class GraphqlRequest(
    @SerializedName("query") val query: String,
    @SerializedName("variables") val variables: HashMap<String, Any>
)
